zubhub_frontend/zubhub/src/views/search_results/SearchResults.jsx

```diff
@@ -256,6 +293,7 @@ function SearchResults(props) {
           </Container>
         ) : (
           <ErrorPage
+            style={{ maxWidth: '100vw' }}
```

Comment:
avoid inline styles and use dvw, it updates if the visible region dimensions change like hiding location bar in browsers when scrolling